src/hydra.ts

```diff
@@ -27,8 +27,9 @@ let lucid = await Lucid.new(undefined, "Preprod");
 
 // Load or generate a session key
 
+let persistentSession = true;
 let privateKey = window.localStorage.getItem("hydra-doom-session-key");
-if (privateKey == null) {
+if (persistentSession && privateKey == null) {
```

Comment:
@Quantumplation There is another place where localstorage is used. I'll set it also below.

Or should we maybe use a `.env` var to only enable this in `DEVELOPMENT` or so?

Reply:
Made it fully optional with: https://github.com/cardano-scaling/hydra-doom/commit/5a123853c3454d79a84ae4f659464349d5dd7452